Makefile

```diff
@@ -1,7 +1,8 @@
 package:
 	# This target should be executed by passing in an argument representing the version of the artifacts we are packaging
 	# For example: make package version=r1
-	docker-compose up --build documentation openapi
```

Comment:
Since openapi relies on documentation completing, we'll run them separately. Even though `openapi` depends on `documentation`, there's no guarantee that documentation finishes. See: https://docs.docker.com/compose/startup-order/